src/util/BalancerHelper.sol

```diff
@@ -0,0 +1,131 @@
+pragma solidity ^0.8.13;
+import "src/util/IVault.sol";
+import "src/util/AbstractHelper.sol";
+
+interface BalancerPool {
+    function getSwapFeePercentage() external view returns(uint);
+}
+
+contract BalancerHelper is AbstractHelper{
+
+    IVault immutable vault;
+    bytes32 immutable poolId;
+    BalancerPool balancerPool;
+    IVault.FundManagement fundManangement;
+
+    constructor(address _dola, address _dbr, address _weth, bytes32 _poolId, address _vault) AbstractHelper(_dola, _dbr, _weth){
+        vault = IVault(_vault);
+        poolId = _poolId;
+        (address balancerPoolAddress,) = vault.getPool(_poolId);
+        balancerPool = BalancerPool(balancerPoolAddress);
+        fundManangement.sender = address(this);
+        fundManangement.fromInternalBalance = false;
+        fundManangement.recipient = payable(address(this));
+        fundManangement.toInternalBalance = false;
+        IERC20(_dola).approve(_vault, type(uint).max);
+        IERC20(_dbr).approve(_vault, type(uint).max);
+    }
+
+    /**
+    @notice Sells an exact amount of DBR for DOLA in a balancer pool
+    @param amount Amount of DBR to sell
+    @param minOut minimum amount of DOLA to receive
+    */
+    function _sellExactDbr(uint amount, uint minOut) internal override {
+        IVault.SingleSwap memory swapStruct;
+
+        //Populate Single Swap struct
+        swapStruct.poolId = poolId;
+        swapStruct.kind = IVault.SwapKind.GIVEN_IN;
+        swapStruct.assetIn = IAsset(address(DBR));
+        swapStruct.assetOut = IAsset(address(DOLA));
+        swapStruct.amount = amount;
+        //swapStruct.userData: User data can be left empty
+
+        vault.swap(swapStruct, fundManangement, minOut, block.timestamp+1);
```

Comment:
Small:
should be unnecessary the +1 to the timestamp since this swap is executed atomically. Same is true for all other places `block.timestamp+1` is used